explorer/app/hooks/useCurrentDetailTab.ts

```diff
@@ -0,0 +1,23 @@
+import { EntityConfig } from "app/config/model";
+import { useRouter } from "next/router";
+
+/**
+ * Hook to get the current tab that will be used to create the detail page
+ * @param currentEntity
+ */
+export const useCurrentDetailTab = (currentEntity?: EntityConfig) => {
+  const router = useRouter();
+  const tabRoute = router.query.uuid?.[1] ?? "";
+
+  if (!currentEntity) {
+    return { currentTab: undefined, tabIndex: -1 };
```

Comment:
What do you think about returning `undefined` here instead of the object? We could then use type narrowing in the callers to determine if the tabs are configured or not.